src/main/java/net/dv8tion/jda/core/handle/MessageReactionHandler.java

```diff
@@ -81,14 +81,14 @@ protected Long handleInternally(JSONObject content)
         {
             channel = api.getPrivateChannelById(channelId);
         }
-        if (channel == null && api.getAccountType() == AccountType.CLIENT)
-        {
-            channel = api.asClient().getGroupById(channelId);
-        }
         if (channel == null)
         {
             channel = api.getFakePrivateChannelMap().get(channelId);
         }
+        if (channel == null && api.getAccountType() == AccountType.CLIENT)
+        {
+            channel = api.asClient().getGroupById(channelId);
+        }
```

Comment:
Why was this moved down?

Reply:
Because I don't see a logical difference and I suspect CLIENT to be the least likely case.